content/workshops/ansible_tower/exercise3.0.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: "Exercise 3.0 - Using Ansible to Implement Security"
+workshops: ansible_tower
+workshop_weight: 30
+layout: lab
+---
+
+:license_url: http://ansible-workshop-bos.redhatgov.io/ansible-license.json
+
+:icons: font
+:imagesdir: /workshops/ansible_tower/images
+
+
+= Exercise 3.0 - Using Ansible to Implement Security
+
+In this exercise, we are going to use Ansible tower to run DISA STIG and NIST 800-53 evaluations of our environment.
+
+NIST 800-53 controls
+https://galaxy.ansible.com/rhtps/800-53/
+
+DISA STIG controls
+https://galaxy.ansible.com/MindPointGroup/RHEL7-STIG/
+
+== Adding the NIST 800-53 role to your tower node
+
+=== Step 1:
+
+In your wetty window (if you closed it, see the link:/workshops/ansible_tower/setup/[SETUP] step, in your workbook), type the following:
+
+[source,bash]
+----
+sudo ansible-galaxy install rhtps.800-53
+----
+
+=== Step 2:
+
+Now that you have the NIST 800-53 role installed on your Tower node, we need to create a playbook that uses it.  First, let's create a storage directory for playbooks, so that we can reference them from a Tower project, and then we'll create the playbook:
+
+[source,bash]
+----
+sudo mkdir -p /var/lib/awx/projects/playbooks
+sudo vim /var/lib/awx/projects/playbooks/800-53.yml
+----
+
+=== Step 3:
+
+Let's create the needed playbook.  You should have a pretty good understanding of playbooks by now, so we won't go over this, in detail.  Please do feel free to ask questions, though, if there is something that you don't understand.
+
+Note the use of the `register`, `with_items`, and `when` directives.  These are how Ansible implements variable creation, recursion, and conditionals, respectively.
```

Comment:
I like to leave a link to the new concepts, i.e. http://docs.ansible.com/ansible/latest/playbooks_conditionals.html#register-variables
http://docs.ansible.com/ansible/latest/playbooks_loops.html#standard-loops
http://docs.ansible.com/ansible/latest/playbooks_conditionals.html#the-when-statement